progsrc/android-template/app/src/main/jni/Android.mk

```diff
@@ -11,10 +11,8 @@ include $(PREBUILT_STATIC_LIBRARY)
 
 include $(CLEAR_VARS)
 
-############ you might need to change this if you move the
-############ android studio project somewhere else, i.e.
-############ outside of the opensmile root folder
-OPENSMILE_ROOT := ../../../../../../
+############ set here the absolute path of your opensmile directory
+OPENSMILE_ROOT := /path/to/opensmile/root/dir
```

Comment:
It successfully run the buildAndroid.sh (`sh buildAndroid.sh`) script using the relative path, but I got a linker error when I tried build the actual android app using gradle (with ` ./gradlew assembleDebug') at the ndk compilation step:

```
make: Entering directory `OPENSMILE_ROOT/progsrc/android-template/app/src/main'
[armeabi] Gdbserver      : [arm-linux-androideabi-4.8] libs/armeabi/gdbserver
[armeabi] Gdbsetup       : libs/armeabi/gdb.setup
[armeabi] Compile++ thumb: smile_jni <= smile_jni.cpp
In file included from jni/smile_jni.cpp:51:0:
jni/smilextract.h:49:32: fatal error: core/smileCommon.hpp: No such file or directory
 #include <core/smileCommon.hpp>
                                ^
compilation terminated.
make: *** [obj/local/armeabi/objs-debug/smile_jni/smile_jni.o] Error 1
make: Leaving directory `OPENSMILE_ROOT/progsrc/android-template/app/src/main'
:app:ndkBuild FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:ndkBuild'.
> Process 'command 'NDK_ROOT/ndk-r10e/ndk-build'' finished with non-zero exit value 2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 10.148 secs
```

it went away when I was using the absolute path (and rerun `buildAndroid.sh`)